[2024-09-17T18:44:42.288+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-17T18:44:42.531+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.extract_data manual__2024-09-17T18:44:41.133855+00:00 [queued]>
[2024-09-17T18:44:42.544+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: NYC_Taxi_pipeline.extract_data manual__2024-09-17T18:44:41.133855+00:00 [queued]>
[2024-09-17T18:44:42.544+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-17T18:44:42.559+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): extract_data> on 2024-09-17 18:44:41.133855+00:00
[2024-09-17T18:44:42.567+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=64) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-17T18:44:42.568+0000] {standard_task_runner.py:72} INFO - Started process 66 to run task
[2024-09-17T18:44:42.568+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'NYC_Taxi_pipeline', 'extract_data', 'manual__2024-09-17T18:44:41.133855+00:00', '--job-id', '196', '--raw', '--subdir', 'DAGS_FOLDER/NYC_Taxi_pipeline.py', '--cfg-path', '/tmp/tmpt697ro8g']
[2024-09-17T18:44:42.570+0000] {standard_task_runner.py:105} INFO - Job 196: Subtask extract_data
[2024-09-17T18:44:42.615+0000] {task_command.py:467} INFO - Running <TaskInstance: NYC_Taxi_pipeline.extract_data manual__2024-09-17T18:44:41.133855+00:00 [running]> on host 103d737d0864
[2024-09-17T18:44:42.702+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mahmoud' AIRFLOW_CTX_DAG_ID='NYC_Taxi_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-17T18:44:41.133855+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-17T18:44:41.133855+00:00'
[2024-09-17T18:44:42.703+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-17T18:44:42.718+0000] {NYC_Taxi_pipeline.py:39} INFO - Data extraction started
[2024-09-17T18:44:42.718+0000] {data_process.py:11} INFO - Loading Parquet files from the folder...
[2024-09-17T18:44:42.719+0000] {data_process.py:42} ERROR - Error while extracting trip data: No objects to concatenate
[2024-09-17T18:44:42.719+0000] {NYC_Taxi_pipeline.py:41} INFO - Data extraction done
[2024-09-17T18:44:42.719+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-17T18:44:42.730+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-17T18:44:42.730+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=NYC_Taxi_pipeline, task_id=extract_data, run_id=manual__2024-09-17T18:44:41.133855+00:00, execution_date=20240917T184441, start_date=20240917T184442, end_date=20240917T184442
[2024-09-17T18:44:42.782+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-17T18:44:42.805+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-17T18:44:42.806+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
